### ISSUE TITLE
Update Inspector to Flutter 3.0

### ISSUE THREAD

**Issue by StarProxima** [kekland/inspector#2]
Hello, inspector 1.1.1 causes errors when building on flutter 3.0. It's a bit unpleasant, could you fix it?
```
/C:/flutter_windows_2.10.4-stable/flutter/.pub-cache/hosted/pub.dartlang.org/inspector-1.1.1/lib/src/inspector.dart:173:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
- 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/flutter_windows_2.10.4-stable/flutter/packages/flutter/lib/src/widgets/binding.dart').
package:flutter/…/widgets/binding.dart:1
      WidgetsBinding.instance?.addPostFrameCallback((_) {
                     ^

/C:/flutter_windows_2.10.4-stable/flutter/.pub-cache/hosted/pub.dartlang.org/inspector-1.1.1/lib/src/widgets/inspector/overlay.dart:39:20: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
- 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/flutter_windows_2.10.4-stable/flutter/packages/flutter/lib/src/widgets/binding.dart').
package:flutter/…/widgets/binding.dart:1
    WidgetsBinding.instance?.scheduleFrameCallback(
                   ^
```

```flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.0.5, on Microsoft Windows [Version 10.0.22000.795], locale ru-RU)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Chrome - develop for the web
[✓] Visual Studio - develop for Windows (Visual Studio Community 2022 17.0.5)
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.70.0)
[✓] Connected device (4 available)
[✓] HTTP Host Availability

• No issues found!

**Comment by kekland** [comment]
@StarProxima should be fixed with version 1.1.4, can you try it out? :)

**Comment by StarProxima** [comment]
> @StarProxima should be fixed with version 1.1.4, can you try it out? :)

Everything works fine, thanks.
